### ISSUE TITLE
Rank issues by Bounty

### ISSUE THREAD

**Issue by usselman** [pow-co/powco.dev#56]
I should be able to rank issues by bounty to find the ones I want to work on.

**Comment by GuiDeMa** [comment]
agreed 100% but right now we are fetching bounty address balance from client side, which is sub optimal. 
Getting balance from the API will allow us to rank by bounty amount more easily
